backend/app/services/adapters/executor_kinds.py

```
@@ -100,16 +103,26 @@ def _get_subtasks_for_task(self, db: Session, task_id: int, status: str, limit:
-        tasks = db.query(Kind).filter(
-            Kind.kind == "Task",
-            Kind.is_active == True,
-            text("JSON_EXTRACT(json, '$.status.status') = :status")
-        ).params(status=status).order_by(
-            Kind.created_at.desc()  # Sort by creation time descending, prioritize latest tasks
-        ).limit(limit).all()
+        tasks = None
+        if type == "offline":
+            tasks = db.query(Kind).filter(
+                Kind.kind == "Task",
+                Kind.is_active == True,
```

Ruff (0.14.2)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)